app/components/Form/SubmitButtons.tsx

```diff
@@ -0,0 +1,108 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+import Button from '@button-inc/bcgov-theme/Button';
+import LoadingSpinner from 'components/LoadingSpinner';
+
+const StyledFlex = styled('div')`
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width: 600px) {
+    flex-direction: row;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  margin: 24px 0;
+  white-space: nowrap;
+  min-width: 178px;
+
+  @media (min-width: 600px) {
+    margin: 0 24px;
+  }
+`;
+
+const StyledToast = styled('div')`
+  background-color: ${(props) => props.theme.color.successGreen};
+  border-radius: 4px;
+  padding: 12px 24px;
+  color: #ffffff;
+  font-size: 16px;
+  transition: all ease-in-out 0.2s 0.2s;
+  text-align: center;
+
+  & a {
+    color: #ffffff;
+  }
+`;
+
+type Props = {
+  disabled: boolean;
+  formData: any;
+  isSubmitPage: boolean;
+  isUpdating: boolean;
+  saveAsDraft: boolean;
+  saveForm: any;
+  status: string;
+};
+
+const SubmitButtons = ({
+  disabled,
+  formData,
+  isSubmitPage,
+  isUpdating,
+  saveAsDraft,
```

Comment:
Same comment as for the `ApplicationForm` state. Here the name is even more problematic, because of the `saveForm` below, which is a function.